src/transformers/modeling_tf_bert.py

```diff
@@ -904,42 +978,44 @@ def call(
             Tokens with indices set to ``-100`` are ignored (masked), the loss is only computed for the tokens with labels
             in ``[0, ..., config.vocab_size]``
         """
-        return_dict = return_dict if return_dict is not None else self.bert.return_dict
         if isinstance(inputs, (tuple, list)):
             labels = inputs[9] if len(inputs) > 9 else labels
+
             if len(inputs) > 9:
                 inputs = inputs[:9]
         elif isinstance(inputs, (dict, BatchEncoding)):
             labels = inputs.pop("labels", labels)
 
         outputs = self.bert(
             inputs,
-            attention_mask=attention_mask,
-            token_type_ids=token_type_ids,
-            position_ids=position_ids,
-            head_mask=head_mask,
-            inputs_embeds=inputs_embeds,
-            output_attentions=output_attentions,
-            output_hidden_states=output_hidden_states,
-            return_dict=return_dict,
-            training=training,
+            attention_mask,
```

Comment:
Same as above, this is not very readable.

Reply:
Well parameters have the exact same name than the variables, I don't see what is not readable ^^

But I can put it back if you want, no problem!

Reply:
Yes but like above, if you tweak the code to experiment or copy it to implement a new model, you have absolutely no idea about the order of the parameters.

Reply:
The documentation is there for that, no? But yes in case of lazy people I can buy this ahah, I will put it back^^